packages/openssh.rb

```diff
@@ -3,39 +3,36 @@
 class Openssh < Package
   description 'OpenSSH is the premier connectivity tool for remote login with the SSH protocol.'
   homepage 'https://www.openssh.com/'
-  version '8.4-1'
+  version '8.6'
   license 'BSD and GPL-2'
   compatibility 'all'
-  source_url 'https://github.com/openssh/openssh-portable/archive/V_8_4_P1.tar.gz'
-  source_sha256 'b924181983a77cb10e61530960e818ccae075e5c457ea6b9f67e4946009563db'
+  source_url 'https://github.com/openssh/openssh-portable/archive/refs/tags/V_8_6_P1.tar.gz'
+  source_sha256 'b641e88a61b58d60f87b1c0e38805eb5b1206810cab71b1f2966faa87938593b'
 
-  binary_url ({
-     aarch64: 'https://downloads.sourceforge.net/project/chromebrew/armv7l/openssh-8.4-1-chromeos-armv7l.tar.xz',
-      armv7l: 'https://downloads.sourceforge.net/project/chromebrew/armv7l/openssh-8.4-1-chromeos-armv7l.tar.xz',
-        i686: 'https://downloads.sourceforge.net/project/chromebrew/i686/openssh-8.4-1-chromeos-i686.tar.xz',
-      x86_64: 'https://downloads.sourceforge.net/project/chromebrew/x86_64/openssh-8.4-1-chromeos-x86_64.tar.xz',
+  binary_url({
+    aarch64: 'https://downloads.sourceforge.net/project/chromebrew/armv7l/openssh-8.6-chromeos-armv7l.tar.xz',
+     armv7l: 'https://downloads.sourceforge.net/project/chromebrew/armv7l/openssh-8.6-chromeos-armv7l.tar.xz',
+       i686: 'https://downloads.sourceforge.net/project/chromebrew/i686/openssh-8.6-chromeos-i686.tar.xz',
+     x86_64: 'https://downloads.sourceforge.net/project/chromebrew/x86_64/openssh-8.6-chromeos-x86_64.tar.xz'
   })
-  binary_sha256 ({
-     aarch64: '6cc250043f9f5978e65220cc5a499651a292f0d8597eedb97e427ccb80872d29',
-      armv7l: '6cc250043f9f5978e65220cc5a499651a292f0d8597eedb97e427ccb80872d29',
-        i686: '80b048feee0b23334ffc90fecba4532139ae3f7ccf7bf8bac525a38709f98436',
-      x86_64: 'e50c359bf9872cb674f90087654d35d3e2d880d3f869c5ed443b179018de1402',
+  binary_sha256({
+    aarch64: '13c4fc76c595404df4a6b53be4e35580bda59937841668c73b4b2f871d559350',
+     armv7l: '13c4fc76c595404df4a6b53be4e35580bda59937841668c73b4b2f871d559350',
+       i686: 'e64aba77b0115d4cd3248961215b2ae638548d40f6b31707202de42792895f11',
+     x86_64: '4900147acf20bfcf9dd27654563a2aeefed7b63e12eb7a9fa96992194ce07f24'
   })
 
   depends_on 'autoconf' => :build
```

Comment:
You could take this out too since buildesential is installed automagically anytime there is a build.